homeassistant/components/greeneye_monitor/__init__.py

```diff
@@ -55,6 +57,12 @@
     }
 )
 
+VOLTAGE_SENSOR_SCHEMA = vol.Schema(
+    {vol.Required(CONF_NUMBER): vol.Range(1, 1), vol.Required(CONF_NAME): cv.string}
```

Comment:
If it is always 1 why does it need to be specified in the configuration?

Reply:
It doesn't really have to be specified, but I thought it made the config more consistent with the other sensors, and it would make it easier to extend in the future if brultech adds more sensors.  I could hardcode it to just 1 and ask only for name instead if you prefer?

Technically, I should probably change it to be 1, 48, as it needs to be a channel number that exists in the GEM, I'm not aware of a way to not have channel 1 currently, but, I guess you never know?

Should I make it 1,48?  Remove the number?

Reply:
IMHO 1,48 is the best option in this case.